plasma_framework/docs/integration-docs/integration-doc.md

```diff
@@ -867,9 +917,98 @@ PaymentExitGame.challengeInFlightExitOutputSpent([
 ])
 ```
 
-## Exit game events
-When listening for events related to the exit game, it's important to remember that there will be only one exit game per transaction type.
+### Delete an in-flight exit
+
+For in-flight exit, an exit is only enqueued to the PlasmaFramework when the exit is piggybacked. As a consequence, if an in-flight exit end up not being piggybacked, the in-flight exit bond would end up being locked.
+
+As a mitigation, we have the `deleteNonPiggybackedInFlightExit` api on the exit game contract. One can call this to retrieve the bond back and clean up the state of in-flight exit. This can only be called after the first phase has been passed and nobody piggybacked the exit.
```

Comment:
```suggestion
As a mitigation, we have the `deleteNonPiggybackedInFlightExit` api on the exit game contract. One can call this to release the bond back to its owner and clean up the state of in-flight exit. This can only be called after the first phase has been passed and nobody has piggybacked the exit.
```